scripts/ioc_deploy.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/python3
+"""
+ioc-deploy is a script for building and deploying ioc tags from github.
+It will create a shallow clone of your IOC in the standard release area at the correct path and "make" it.
+"""
+
+import argparse
+import enum
+import logging
+import os
+import subprocess
+import sys
+import urllib.error
+import urllib.request
+from pathlib import Path
+
+EPICS_SITE_TOP_DEFAULT = "/cds/group/pcds/epics"
+GITHUB_ORG_DEFAULT = "pcdshub"
+
+logger = logging.getLogger("ioc-deploy")
+
+
+if sys.version_info >= (3, 7, 0):
```

Comment:
Could we activate pcds-conda before runnign this in the `ioc-deplpy` script instead of supporting old python?

Do we expect pcds-conda to not be available in situations where this script is run?

Reply:
I didn't want to spend the few seconds it takes to activate the env if it wasn't needed.